docs/docs/actions.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Actions"
+disableTableOfContents: true
+jsdoc: "/gatsby[/]src[/]redux[/]actions[/](public|restricted)[.]js$/"
```

Comment:
We're going to move these definitions somewhere easier to pull from (or even turn them into typedefs?) #19383. I think we might be able to simplify it once we do that?

Reply:
It looks like file path is going to be what determines which language, so these should refer to *specific* files. My thought is something like this would be ideal. This way the queries would only need to be modified to look the correct place.
```yaml
jsdoc:  ["gatsby/src/redux/actions/public.js","gatsby/src/redux/actions/restricted.js"]
```

Reply:
are multiple allowed?

```yaml
jsdoc: "gatsby/src/redux/actions/public.js"
jsdoc: "gatsby/src/redux/actions/restricted.js"
```

Reply:
I don't think that is proper YAML (if such a thing even exists). It is pretty nice about list syntax though, another equivalent notation would be:
```yaml
jsdoc:  
  - "gatsby/src/redux/actions/public.js"
  - "gatsby/src/redux/actions/restricted.js"
```

Reply:
Ah. Frontmatter is yaml. Nice to know. I did not found a technical spec about frontmatter :(

Reply:
somehow I remember I had seen the double tags somewhere. But I am not sure

Reply:
My learning moment was "why did a : break everything?" It really depends on the parser I think. Gatsby puts it into a pretty unforgiving GraphQL schema, so switching from string to list probably would make the whole thing blow up.

Reply:
My learning moment was "why did a : break everything?" It really depends on the parser I think. Gatsby puts it into a pretty unforgiving GraphQL schema, so switching from string to list probably would make the whole thing blow up.